### ISSUE TITLE
不确定性融合

### ISSUE THREAD

**Issue by Auditor1234** [lblaoke/TLC#10]
你好，请问论文中不确定性融合、对证据加权中的前缀权重两个公式是在本文中创新性地提出来的吗，还是参考了已有的方法呢

**Comment by lblaoke** [comment]
Eq. 7 与 Eq. 8 为本文原创，但其形式的灵感来自 [1] 中的 Eq. 4

[1] Trusted Multi-View Classification. ICLR 2020.

**Comment by Auditor1234** [comment]
非常感谢你的回答，不过还有一些问题，在看项目代码时，对于model/loss.py文件有些疑问，在forward()函数末尾求loss的时候，下面这段代码貌似是把`w > tau`的那些样本损失算了两遍，而其他的算了一遍，而不是用指示函数直接舍去那些`w`较低的样本，是这样吗
```
# dynamic engagement
w = extra_info['w'][i]/extra_info['w'][i].max()
w = torch.where(w>self.tau,True,False)
loss += (w*l).sum()/w.sum()
loss += l.mean()
```

**Comment by lblaoke** [comment]
此处是笔误，应该是
```
l = (w*l).sum()/w.sum()
loss += l.mean()
```
感谢指出这一问题！

**Comment by Auditor1234** [comment]
好的，感谢解答
